src/components/unique-token/UniqueTokenCard.js

```diff
@@ -64,9 +59,13 @@ const UniqueTokenCard = ({
           backgroundColor={item.background || colors.lightestGrey}
           imageUrl={item.lowResUrl || item.image_url}
           isCard
-          item={item}
-          resizeMode={resizeMode}
-          small={smallENSName}
+          fullUniqueId={item.fullUniqueId}
+          id={item.id}
+          collectionName={item.collection.name}
+          name={item.name}
+          uniqueId={item.uniqueId}
+          mimeType={item.mimeType}
+          address={item.asset_contract.address}
```

Comment:
Thought this crash was fixed initially, but I think I found a better way to repro.

https://www.loom.com/share/7112b9d6664249698351b9d873240b67

If you don't change sort and stick the the default 'recent' sort it doesn't happen, but if you change sort then navigate to the ENS Profile Sheet, you can repro the crash. I'm guessing this is caused by an underlying existing issue, but is made more apparently because we're not optionally accessing `item.collection.name` and `item.asset_contract.address` in this new change.